src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/armor/StomperBoots.java

```diff
@@ -89,4 +91,20 @@ private Vector getShockwave(Location origin, Location target) {
         }
     }
 
+    /**
+     * Checks if the stomper boots can move an entity and is not the player who is using the boots.
+     * <br><b>For developers:</b> If you're spawning an immovable NPC, you should be denying
+     * collision with {@code LivingEntity#setCollidable(false)} or
+     * gravity with {@code LivingEntity#setGravity(false)}.
+     *
+     * @param entity
+     *            The {@link LivingEntity} to check.
+     * @param player
+     *            The {@link Player} using the {@link StomperBoots}.
+     * @return If the entity can move.
+     */
+    private boolean canMoveEntity(@Nonnull LivingEntity entity, @Nonnull Player player) {
```

Comment:
I would suggest to swap the order of parameters here and rename the method.
The current order is unintuitive in my opinion or at least in a grammatical sense. If the method was called "canBePushedBy" then the current order would make more sense. So I am in favour of swapping these here.

Also if you want this to be overridden, then it should be at least `protected`.

```suggestion
    protected boolean canPush(@Nonnull Player player, @Nonnull LivingEntity entity) {
```